server/base/src/main/java/org/apache/accumulo/server/util/ChangeSecret.java

```diff
@@ -63,12 +66,15 @@ public static void main(String[] args) throws Exception {
     Instance inst = opts.getInstance();
     if (!verifyAccumuloIsDown(inst, opts.oldPass))
       System.exit(-1);
-    String instanceId = rewriteZooKeeperInstance(inst, opts.oldPass, opts.newPass);
-    updateHdfs(fs, inst, instanceId);
+    if (!verifyHdfsWritePermission(fs))
+      System.exit(-1);
```

Comment:
Might be better to print a message first.

Also (and this is perhaps a minor nit), negative exit codes should not be used. They aren't cross-platform. Also, the special exit codes defined in sysexits.h should be avoided, so best to stick with values `[1,63]`. I'd probably just stick with `1` for this (and the other places in this class, which currently use `-1`).